src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import core.basesyntax.enums.Colors;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int DEFAULT_RADIUS = 10;
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        Figure[] figures = new Figure[] {new Circle(), new Square(), new Rectangle(),
+                new RightTriangle(), new IsoscelesTrapezoid()};

```

Comment:
should we create instances of all figures if we want to return only one of them?